docs/tutorials/fundamentals/viewer.md

```diff
@@ -404,40 +379,53 @@ viewer.camera.zoom = 2
 viewer.camera.angles = (3, 38, 53)
 nbscreenshot(viewer, alt_text="A rotated 3D view")
 ```
+![image: Rotated image](../assets/tutorials/rotated-image.png)  
 
-Next to the 2D / 3D button is a button to roll the dimensions that are currently
-being displayed in the viewer - for example if you have a `ZYX` volume and are
-looking at the `YX` slice this will then show you the `ZY` slice.
+The third button rolls the dimensions that are currently displayed in the viewer. For example if you have a `ZYX` volume and are looking at the `YX` slice, this will then show you the `ZY` slice.
 
-After that is a button that transposes the displayed dimensions.
+The fourth button transposes the displayed dimensions.
 
-After that is a button the toggles grid mode.
+Then there is a grid button that toggles grid mode. When clicked it displays each layer of the image in its own tile. 
 
-Finally, there is the `home` button that will reset the camera state to its
-initial values.
+Finally, there is the `home` button. It resets the camera to its initial values.
 
-+++
 
 ### Status bar
 
-At the very bottom of the GUI there is a status bar that contains useful updates
-and tips.
-
-On the lefthand side of the status bar there is a message that contains
-information about the position of the mouse and the values of any images or the
-indices of any points that are currently hovered over, depending on which layer
-is selected. The status bar will also display information about what button you
-are clicking in the layer control panel too.
-
-The righthand side of the status bar contains some helpful tips depending on
-which layer and tools are currently selected.
-
-+++
-
-## Changing viewer theme
-
-Currently, **napari** comes with two different themes `light` and `dark`, which
-is the default. To change the theme, update `theme` property of the viewer:
+At the very bottom of the GUI there is a status bar that contains useful updates and tips.
+
+On the left side of the status bar there is a message about the position of the mouse and the values of any images or the indices of any points that are currently hovered over, depending on which layer is selected. When there are buttons in the layer controls panel, the status bar displays information about the layer controls button you are clicking. The buttons are not available for every layer type.  
+
+The right side of the status bar contains some helpful tips depending on which layer and tools are currently selected.
+
+## Right-click menu
```

Comment:
I think it would be best to rearrange this section to match the way these items are presented in the menu?
<img width="221" alt="image" src="https://user-images.githubusercontent.com/76622105/208297694-d1d41915-d8b0-4916-984a-393a62e04cf9.png">

Reply:
Bump, any thoughts on this? the current ordering seems random?

Reply:
The options that appear on the right click menu depend on what type of layer is selected.  I could put these all in alphabetical order?